src/components/navbar/navbar.css

```diff
@@ -0,0 +1,14 @@
+.navbar{
+    height: 4rem;
+}
+
+.nav-menu{
+    position: absolute;
+    z-index: 10;
```

Comment:
Try using immediate next values for z-indexes and not to skip z-index values.
So, IMHO z-index: 2 or 3 would be a better approach.

Reply:
will do that